RFS/src/main/java/com/rfs/ReindexFromSnapshot.java

```diff
@@ -103,19 +116,50 @@ public static void main(String[] args) {
             throw new IllegalArgumentException("Unsupported target version: " + sourceVersion);
         }
 
+        /*
+         * You have three options for providing the snapshot data
+         * 1. A local snapshot directory
+         * 2. A source host we'll take the snapshot from
```

Comment:
Another use case to potentially add in future, seems like would be to allow taking a local snapshot and using

Reply:
Potentially; I thought about that as a option and wasn't able to think of a situation where I'd need it for testing but if you have something in mind I'm all ears.

Reply:
My mind was around setting up a docker compose for local testing that had a low barrier for entry using a local snapshot. I'm worried having to setup an S3 bucket might turn some users away from initial trying the tool.

Reply:
Got it - yep, I could see that.